fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/expressions/LogicalSortExpression.java

```diff
@@ -109,6 +119,22 @@ public Value getResultValue() {
         return inner.getFlowedObjectValue();
     }
 
+    @Override
+    public boolean semanticEquals(@Nullable final Object other, @Nonnull final AliasMap aliasMap) {
+        if (!RelationalExpressionWithChildren.super.semanticEquals(other, aliasMap)) {
+            return false;
+        }
+        LogicalSortExpression otherSort = (LogicalSortExpression)other;
+        // TODO: need semanticEquals on the Values?
+        return this.ordering.equals(otherSort.ordering);
+    }
+
+    @Override
+    public int semanticHashCode() {
```

Comment:
You probably don't want to override this one, but `hashCodeWithoutChildren()`.

Reply:
Removed